integration-tests/cdi-app/cdi-app-standalone/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>jsf-app</artifactId>
```

Comment:
@Theoderich the build failed with:
```
[2019-08-14T06:14:27.054Z] [ERROR] [ERROR] Some problems were encountered while processing the POMs:

[2019-08-14T06:14:27.054Z] [FATAL] Non-resolvable parent POM for co.elastic.apm:cdi-app-standalone:[unknown-version]: Could not find artifact co.elastic.apm:jsf-app:pom:1.8.1-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 5, column 13
```
Is this parent reference valid with the path of the module and pom?